pm2-windows.json

```diff
@@ -1,14 +1,25 @@
 {
   "apps": [
+   {
+      "name": "@ove/ove-asset-manager",
+      "cwd": "./packages/ove-asset-manager",
+      "kill_timeout": 3000,
+      "restart_delay": 3000,
+      "script": "dotnet",
+      "args": "dist/OVE.Service.ImageTiles.dll --server.urls=http://0.0.0.0:8181",
+      "env": {
+          "PORT": "8181"
+      }
+    },
     {
       "name": "@ove/ove-service-imagetiles",
```

Comment:
Is not Asset manager a replacement for ImageTiles rather than addition? If so, would this not break when you attempt to use PM2?

Reply:
ImageTileService will depend upon the Asset Manager Service

Reply:
@davidbirchwork, I raised an issue on the same point, without seeing this before. I believe @DavidAkroyd is pointing out that the pm2-windows.json does not seem to work based on how you have configured it.

Reply:
will fix in next commit